styles/styles.css

```diff
@@ -0,0 +1,1844 @@
+  }
+  footer p {
+    text-align: center;
+    font-size: 1rem;
+    line-height: 1rem;
+  }
+  .footer-logo {
+    max-height: 5rem;
+    font-weight: bold;
+  }
+  .nav-footer {
+    display: flex;
+    flex-direction: row;
+  }
+  .nav-item-ft {
+    color: #ffffff;
+    text-decoration: none;
+  }
+}
+/*XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX*/
+/*--Media Screen 768px tablet Phone--*/
+/*XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX*/
+@media only screen and (min-width: 768px) {
+  body {
+    width: 100%;
+    padding: 0;
+    margin: 0;
+  }
+  header {
+    display: flex;
+    background-image: url(../images/header-image.jpg);
+    background-size:auto;
+    height: 15rem;
+    width: auto;
+    /*background: linear-gradient(to right, #A52A2A, #A9A9A9);*/
+  }
+  .dog-img-header p {
+    padding: 0;
+    font-size: 1rem;
+    margin-left: 2rem;
+    width: 200%;
+  }
+  .dog-img-header {
+    width: 500%;
+    height: 40%;
+    padding-left: 1rem;
+    align-items: flex-start;
+  }
+  .svg-dog-header {
+    display: flex;
+    width: 10rem;
+  }
+  .dog-img-header {
+    width: 20%;
+    padding-top: 5rem;
+  }
+  .svg-dog-header p {
+    font-size: inherit;
+    width: 200%;
+  }
+  .nav-head {
+    display: flex;
+    flex-direction: row;
+
+  }
+  .nav-item-head {
+    color:#2f4f4f;
+    font-weight: 600;
+    text-decoration: none;
+    font-size: 1rem;
+    font-family:Verdana, Geneva, Tahoma, sans-serif;
+  }
+  .title-about {
+    width: 100%;
+    color: #a9a9a9;
+    font-size: 1.2rem;
+    padding-top: 1rem;
+    padding-bottom: 1rem;
+    line-height: 1.5rem;
+    justify-content: space-evenly;
+  }
+  .title-about p {
+    color: #000000;
+    justify-content: center;
+    align-items: center;
+    width: 84%;
+    font-size: 1.2rem;
+    line-height: 1.8rem;
+  }
+  .title-index {
+    color:#2f4f4f;
+    font-weight: 700;
+    font-size: 2rem;
+    font-family:Verdana, Geneva, Tahoma, sans-serif;
+    padding-left: .8rem;
+  }
+  .blog-adoptadog-h1 {
+    color:#2f4f4f;
+    padding-left: 0;
+    font-size: 1rem;
+    margin-left: 40rem;
+  }
+  .dog-blog-aside {
+    display: none;
+  }
+  .blog-topic {
+    color:#2f4f4f;
+    width: 100%;
+    margin-top: 0;
+    padding-top: 0;
+    font-size: 1.2rem;
+  }
+  .blog-content {
+    display: flex;
+    flex-direction: column;
+    width: 60%;
+    height: 15rem;
+    padding-left: 2rem;
+    padding-bottom: 2rem;
+    padding-top: 0;
+    margin: 0;
+  }
+  .blog-topic {
+    color:#2f4f4f;
+    width: 100%;
+    margin-top: 0;
+    padding-top: 0;
+  }
+  .blog-text-info {
+    color: #000000;
+    width: 100%;
+    margin: 0;
+  }
+  .main-index {
+    display: flex;
+    width: 100%;
+  }
+  .dog-listing-container {
+    width: 100%; 
+  }
+  .adopt-a-dog-intro {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    margin: 0 3rem 0 3rem;
+  }
+  .index-image-top {
+    display: flex;
+    align-items: flex-start;
+    width: 80%;
+    height: 20rem;
```

Comment:
Notice how your image seems to get stretched out on larger screen sizes? Try setting a width, such as your 20rem. Then set the height to auto: `height: auto;` this will allow the image to scale but maintain its aspect ratio